functions/New-DbaDbUser.ps1

```diff
@@ -0,0 +1,247 @@
+function New-DbaDbUser {
+	<#
+		.SYNOPSIS
+			Creates a new user for the specified database.
+		.DESCRIPTION
+			Created a new user for a specified database with provided specifications.
+		.PARAMETER SqlInstance
+			The target SQL Server instance. Defaults to the default instance on localhost.
+		.PARAMETER SqlCredential
+			Allows you to login to servers using SQL Logins instead of Windows Authentication (AKA Integrated or Trusted). To use:
+
+			$scred = Get-Credential, then pass $scred object to the -SqlCredential parameter.
+
+			Windows Authentication will be used if SqlCredential is not specified. SQL Server does not accept Windows credentials being passed as credentials.
+
+			To connect to SQL Server as a different Windows user, run PowerShell as that user.
+		.PARAMETER Database
+			Specifies the database(s) to process. Options for this list are auto-populated from the server. If unspecified, all databases will be processed.
+		.PARAMETER ExcludeDatabase
+			Specifies the database(s) to exclude from processing. Options for this list are auto-populated from the server. By default, system databases are excluded.
+		.PARAMETER IncludeSystem
+			If this switch is enabled, the user will be added to system databases.
+		.PARAMETER Login
+			When specified, the user will be associated to this SQL login and have the same name as the Login. Username or Login may be specified but not both.
+		.PARAMETER Username
+			When specified, the user will have this name. Username or Login may be specified but not both.
```

Comment:
Without looking at the code, just based on help info...

Does this mean then that I can't add a Windows Login and map it to a different username within a given database? SQL Server allows the username in the database to be different than the login it is mapped to.